### ISSUE TITLE
Whitespace between a conditional option and the line tag doesn't compile

### ISSUE THREAD

**Issue by EddieCameron** [YarnSpinnerTool/YarnSpinner#220]
Appeared some time between 1.03 and 1.1

This compiles:
```
-> Option A <<if true>>#line:0e8a7ce
-> Option B #line:0405c66
```

But this doesn't (extraneous input):
```
-> Option A <<if true>> #line:0e8a7ce
-> Option B #line:0405c66
```

Easy to workaround for now, but annoying!

**Comment by McJones** [comment]
Bug confirmed and a patch should be out soon, thanks for catching this.

**Comment by desplesda** [comment]
Fixed in 24500a6, will be fixed in final v1.1 release.
